### PR TITLE
Correct typo and file name of HPCS/Key protect design doc

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Review by Madhu-1 — DISMISSED** [review]

File: docs/design/proposals/encrypted-with-keyprotect.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ service from the CSI driver and to make use of the encryption operations:
 The Key Protect/HPCS connection URL.
 
 * IBM_KP_TOKEN_URL
-The Token Authenticaltion URL of KeyProtect/HPCS service.
```
Comment (Madhu-1) [inline docs/design/proposals/encrypted-with-keyprotect.md:19]: better to name the file as `encryption-with-keyprotect`?

Comment (humblec) [reply docs/design/proposals/encrypted-with-keyprotect.md:19]: yeah, kept that way.

File: docs/design/proposals/encryption-with-keyprotect.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ config map to `KMS_SERVICE_NAME`.
 
 ## Volume Encrypt or Decrypt Operation
 
-The IBM Key protect server's `wrap` and `unwrap` functionalities will be used by
+The IBM Key Protect server's `wrap` and `unwrap` functionalities will be used by
```
Comment (Madhu-1) [inline docs/design/proposals/encryption-with-keyprotect.md:59]: not only protect, but Key also need to be capitalized in all places as per https://cloud.ibm.com/docs/key-protect?

Comment (humblec) [reply docs/design/proposals/encryption-with-keyprotect.md:59]: yep, done

**Review by Madhu-1 — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Comment by humblec** [comment]
@mergifyio rebase

**Comment by humblec** [comment]
@mergifyio refresh

**Comment by humblec** [comment]
@Mergifyio refresh

**Comment by humblec** [comment]
@Mergifyio rebase

**Comment by humblec** [comment]
@Mergifyio refresh

**Pull request merged**